Cloud_Region_Metadata_specification.md

```diff
@@ -21,7 +21,7 @@ Core technical scope includes standardized disclosure of location-based and mark
 - **Cloud Region Metadata:**
    - Define standard parameters for cloud region metadata, including cloud provider and region specifications.
    - Establish guidelines for annual updates and data lag management (6-18 months), with emphasis on specifying the year or using the latest available data.
-   - Clarify the annual average location-based marginal grid-carbon intensity value for SCI-o and its availability and handling of not-available (NA) data.
+   - Clarify the region specific carbon intensity value for SCI-o and its availability and handling of not-available (NA) data.
```

Comment:
Based on the SCI, the following should be quoted.

Region-specific carbon intensity factors measure the grid carbon intensity of electricity in a grid region. If the electricity consumption is connected to a grid, the short run marginal, long run marginal, or average emissions grid intensity of that grid shall be used, which excludes any [market-based measures](https://sci.greensoftware.foundation/#market-based-measures).

Reply:
@mrchrisadams - yes please, could you add what I've added 

Region-specific carbon intensity factors measure the grid carbon intensity of electricity in a grid region. If the electricity consumption is connected to a grid, the short-run marginal, long-run marginal, or average emissions grid intensity of that grid shall be used, which excludes any market-based measures.

this to the metadata page